sdk-core/src/main/java/care/data4life/sdk/tag/TagEncryptionService.kt

```diff
@@ -50,77 +66,76 @@ class TagEncryptionService @JvmOverloads constructor(
         return Observable
                 .fromIterable(encryptedList)
                 .map { encryptedTag -> decryptTag(tek, encryptedTag).blockingGet() }
+                .map { encodedTag -> tagHelper.decode(encodedTag) }
                 .filter { decryptedItem -> condition(decryptedItem) }
                 .toList()
                 .map { decryptedList -> transform(decryptedList) }
                 .blockingGet()
     }
 
+    @Throws(D4LException::class)
+    private fun decryptTag(key: GCKey, base64tag: String): Single<String> {
```

Comment:
I would move the private members after the public accessors

Reply:
No problem...For me it's a bit easier to read it this way.